packs/journals/classes.json

```diff
@@ -34,7 +34,7 @@
xpertise <span style=\"float:right\">Level 3</span></h2><p>@Localize[PF2E.SpecificRule.ClassFeatures.FortitudeExpertise.animist.Description]</p><h2>General Feats <span style=\"float:right\">Level 3</span></h2><p>At 3rd level and every 4 levels thereafter, you gain a general feat.</p><h2>Skill Increases <span style=\"float:right\">Level 3</span></h2><p>At 3rd level and every 2 levels thereafter, you gain a skill increase. You can use this increase to either become trained in one skill you're untrained in, or to become an expert in one skill in which you're already trained.</p><p>At 7th level, you can use skill increases to become a master in a skill in which you're already an expert, and at 15th level, you can use them to become legendary in a skill in which you're already a master.</p><h2>Ancestry Feats <span style=\"float:right\">Level 5</span></h2><p>In addition to the ancestry feat you started with, you gain an ancestry feat at 5th level and every 4 levels thereafter.</p><h2>Expert Spellcaster <span style=\"float:right\">Level 7</span></h2><p>The magic of the spirits flows freely through you. Your proficiency ranks for spell attack modifier and spell DC increase to expert.</p><h2>Third Apparition <span style=\"float:right\">Level 7</span></h2><p>@Embed[Compendium.pf2e.classfeatures.Item.bRAjde9LlavcOUuM inline hr=false]</p><h2>Perception Expertise <span style=\"float:right\">Level 9</span></h2><p>@Embed[Compendium.pf2e.classfeatures.Item.JCqACxgrm5ixX0Jy inline hr=false]</p><h2>Expert Protections <span style=\"float:right\">Level 11</span></h2><p>The magic of the spirits flows freely through you. Your proficiency ranks for spell attack modifier and spell DC increase to expert.</p><h2>Simple Weapon Expertise <span style=\"float:right\">Level 11</span></h2><p>@Localize[PF2E.SpecificRule.ClassFeatures.SimpleWeaponExpertise.animist.Description]</p><h2>Master of Mind and Spirit <span style=\"float:right\">Level 13</span></h2><p>@Embed[Compendium.pf2e.classfeatures.Item.8LhwKWBxF2lgxEG5 inline hr=false]</p><h2>Weapon Specialization <span style=\"float:right\">Level 13</span></h2><p>@Embed[Compendium.pf2e.classfeatures.Item.9EqIasqfI8YIM3Pt inline hr=false]</p><h2>Fourth Apparition <span style=\"float:right\">Level 15</span></h2><p>@Embed[Compendium.pf2e.classfeatures.Item.avLo2Jl3mNWssp0W inline hr=false]</p><h2>Master Spellcaster <span style=\"float:right\">Level 15</span></h2><p>Your affinity with the apparitions you bond to has further enhanced the potency of your magic. Your proficiency ranks for spell attack modifier and spell DC increase to master.</p><h2>Legendary Spellcaster <span style=\"float:right\">Level 19</span></h2><p>You are a near-perfect channel for the magical energies of the realms beyond. Your proficiency ranks for spell attack modifier and spell DC increase to legendary.</p><h2>Supreme Incarnation <span style=\"float:right\">Level 19</span></h2><p>@Embed[Compendium.pf2e.classfeatures.Item.1MHXjNczVZfVvDP6 inline hr=false]</p>",
```

Comment:
Did you intend to remove the Embed from Animist?

Reply:
What the?
Hold on